### DISCUSSION TITLE
Detailed "Global Ranking" in result screens

### DISCUSSION THREAD

**Discussion by Asiskan** [ppy/osu#28023] [Ideas]
Maybe just a matter of opinion, but it definitely felt better in stable to see the whole (detailed) number change when submitting a play that would increase/decreased your global ranking instead of it being rounded up/down 
![image](https://github.com/ppy/osu/assets/80349084/006274b0-e417-49b5-8e61-cd0476b7e8a9)

**Comment by peppy** [comment]
Can agree with this one.

**Reply by bdach** [reply]
This was only a thing because some other screens already did this "1.1k" thing. If we're dropping these kinds of units it should arguably happen everywhere (including leaderboard in song select etc.)

**Reply by Joehuu** [reply]
Leaderboard in song select has limited space, while results screen does not. https://github.com/ppy/osu/issues/27296 is another case with limited space, where truncating the rank (at least > 1 mill as shown in the issue) with a full tooltip is feasible.

**Reply by bdach** [reply]
I don't buy the "limited space" argument. If people want to be seeing specific rank numbers then we should make the space to display the full rank.

**Reply by peppy** [reply]
i'd want to show the full numbers everywhere, yeah.
